x/cronos/keeper/precompiles/bank.go

```diff
@@ -53,8 +56,21 @@ func (bc *BankContract) Address() common.Address {
 
 // RequiredGas calculates the contract gas use
 func (bc *BankContract) RequiredGas(input []byte) uint64 {
-	// TODO estimate required gas
-	return BankContractRequiredGas
+	methodID := input[:4]
```

Comment:
nit: I think precompute the map from methodID to gas would be more efficient

Reply:
done